src/theme/index.tsx

```diff
@@ -487,7 +487,23 @@ body {
   }
 }
 
-
+@keyframes loading-rotations-2 {
```

Comment:
why this styles?

Reply:
Because the chain I have has only 2 dexes and this allows us to see the animations in that case.
It solves the 1st issue raised by niemann

Reply:
thanks for the info!

Reply:
This animation is becoming a lot to maintain. When @wixzi worked on it, I anticitpated this long-term. We were in rush. 

This should be refactored into a component that accepts any number of logos and animates them.

Reply:
I agree but I'm not sure we should do it in this PR. I'll create a ticket to rework this.